pkg/util/fs_util.go

```diff
@@ -155,21 +155,24 @@ func ChildDirInWhitelist(path, directory string) bool {
 	return false
 }
 
-func unTar(r io.Reader, dest string) error {
+// unTar returns a list of files extracted from the tar archive
```

Comment:
would be good to including in this docstring what the params `r` and `dest` are for (https://blog.golang.org/godoc-documenting-go-code) e.g. `unTar returns a list of files that have been extracted from the tar archive at r to the path at dest`

Reply:
for sure, just added it!